extensions/wikia/Search/classes/Services/CombinedSearchService.php

```diff
@@ -398,17 +484,14 @@ protected function getImage( $wikiId, $articleId ) {
 			if ( $row ) {
 				$dbName = $row->city_dbname;
 				if ( !empty( $dbName ) ) {
-					$db = wfGetDB( DB_SLAVE, [ ], $dbName ); // throws if database does not exits.
+					$db = wfGetDB( DB_SLAVE, [], $dbName ); // throws if database does not exits.
 					$imageServing = new \ImageServing(
-						[ $articleId ],
-						self::IMAGE_SIZE,
-						self::IMAGE_SIZE,
-						$db
+						[ $articleId ], self::IMAGE_SIZE, self::IMAGE_SIZE, $db
 					);
 					$isResult = $imageServing->getImages( 1 );
-					$images = isset( $isResult[ $articleId ] ) ? $isResult[ $articleId ] : false;
+					$images = isset( $isResult[$articleId] ) ? $isResult[$articleId] : false;
 					if ( $images && sizeof( $images ) > 0 ) {
-						$imageName = $images[ 0 ][ 'name' ];
+						$imageName = $images[0]['name'];
 						$file = \GlobalFile::newFromText( $imageName, $wikiId );
 						if ( $file->exists() ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)